consensus/quorum/quorum.go

```diff
@@ -81,7 +81,8 @@ type ParticipantTracker interface {
 // SignatoryTracker ..
 type SignatoryTracker interface {
 	ParticipantTracker
-	SubmitVote(
+	// This func shouldn't be called directly from outside of quorum. Use AddNewVote instead.
+	submitVote(
```

Comment:
Consider removing from interface?

Reply:
it's used as typing for different type of structs. So not easy to remove.